com.simulide.simulide.yml

```diff
@@ -0,0 +1,30 @@
+id: com.simulide.simulide
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+command: simulide
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=home
+modules:
+  - name: simulide
+    buildsystem: qmake
+    subdir: build_XX
+    build-commands:
+      - install -Dm755 -t /app/bin/ ./executables/SimulIDE_1.0.0-SR2/simulide
+      - install -d /app/share/simulide/
+      - cp -r ./executables/SimulIDE_1.0.0-SR2/data /app/share/simulide
+      - cp -r ./executables/SimulIDE_1.0.0-SR2/examples /app/share/simulide
+      - install -d /app/share/simulide/translations/
+      - cp ../resources/translations/*.qm /app/share/simulide/translations/
+    sources:
+      - type: bzr
+        url: lp:simulide/1.0.0
```

Comment:
you need a revision.

Reply:
also is this a valid repo URL?

Reply:
The developer added the xml file recently, so it's not yet on the zip. I included it temporarily on this repository and I will change it back when the next version is released.